corpus/convert.py

```diff
@@ -177,6 +192,30 @@ def run(self):
                     else segment.recording.name
                 )
                 segment_track = segment.track + 1 if segment.track else 1
+
+                orth = f" {segment.orth.strip()} "
+
+                if self.exclude_non_speech:
+                    for nst in self.non_speech_tokens:
+
+                        def replace_recursive(orthography, token):
+                            pos = orthography.find(f" {token} ")
+                            if pos == -1:
+                                return orthography
+                            else:
+                                orthography = orthography.replace(f" {token} ", "   ")
```

Comment:
why replace with three spaces instead of one?
```suggestion
                                orthography = orthography.replace(f" {token} ", " ")
```

Reply:
I am not sure anymore.. not sure if this has a valid reason..